src/components/BookList.jsx

```diff
@@ -20,28 +35,114 @@ function BookList() {
   }
 
   return (
-    <ul>
+    <ul
+      style={{
+        padding: '0 5em',
+        overflow: 'auto',
+        maxHeight: '265px',
+      }}
+    >
       {Object.entries(books).map(([id, book]) => book.map((bookItem) => (
-        <li key={id}>
+        <li
+          key={id}
+          style={{
+            background: '#fff',
+            padding: '1em',
+            marginBottom: '1em',
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+          }}
```

Comment:
- [ ] I can see you have react inline styling in your app, nice one. However, I will encourage you to pass it as a variable as it is the only acceptable and professional way of passing inline comments just as the example below:

```
const myComponentStyle = {
  color: "blue",
  lineHeight: 10,
  padding: "1.5em",
}

function MyComponent() {
  return <div style={myComponentStyle}> Inline Styled Component</div>
}
```